frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToParentStatusDependentAction.java

```diff
@@ -88,12 +79,9 @@ default void syncToParentStatusDependentActionUpdate(AnActionEvent anActionEvent
     var isStatusEligible = getEligibleStatuses().contains(syncToParentStatus);
 
     if (isStatusEligible) {
-      if (getCurrentBranchNameIfManaged(anActionEvent).equals(branchName)) {
-        presentation.setText(getActionTextForCurrentBranch());
-      }
-
       var parentName = gitMacheteNonRootBranch.getParentBranch().getName();
-      var enabledDesc = format(getEnabledDescriptionFormat(), parentName, branchName.get());
+      // At this point `branchName` must be present, so `.getOrNull()` is here only to satisfy checker framework
```

Comment:
Nit: you can instead start with `var branchName = getNameOfBranchUnderAction(anActionEvent).getOrNull();` and then follow suit... but not insisting on that

Reply:
Actually, I'll change this on a separate PR, no action required here

Reply:
I'd stay with `Option`, it's nice to have this in other places like `gitMacheteBranch` var instantiation